### PR TITLE
ordinary least squares regression

### PR DESCRIPTION
**Pull request by daikiakiyoshi**

### CODE REVIEW

File: experiments/sample/Advertising.csv [hunk 1]
```diff
@@ -0,0 +1,201 @@
+,TV,radio,newspaper,sales
+1,230.1,37.8,69.2,22.1
```
Comment (nishidadaishiro) [inline experiments/sample/Advertising.csv:2]: Let's create a data folder and put sample datasets in there

File: experiments/sample/Sample.ipynb [hunk 2]
```diff
@@ -10,17 +10,20 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.6"
+   "version": "2.7.16"
```
Comment (nishidadaishiro) [inline experiments/sample/Sample.ipynb:13]: rename sample to ols?

File: experiments/sample/Advertising.csv (hunk 1 above)
Comment (daikiakiyoshi) [reply experiments/sample/Advertising.csv]: updated

Comment (daikiakiyoshi) [reply experiments/sample/Advertising.csv]: done

File: experiments/sample/Sample.ipynb (hunk 2 above)
Comment (daikiakiyoshi) [reply experiments/sample/Sample.ipynb]: done

File: experiments/sample/Advertising.csv (hunk 1 above)
Comment (nishidadaishiro) [reply experiments/sample/Advertising.csv]: Did you forget to add the data folder?

File: experiments/sample/Sample.ipynb (hunk 2 above)
Comment (nishidadaishiro) [reply experiments/sample/Sample.ipynb]: Maybe rename the folder also

**Review by nishidadaishiro — APPROVED** [review]

**Pull request merged**